test/Serializers/Tests/Phpunit/DispatchingSerializerTest.php

```diff
@@ -27,7 +27,7 @@ public function testConstructWithNoSerializers() {
 
 	public function testConstructWithInvalidArgumentsCausesException() {
 		$this->setExpectedException( 'InvalidArgumentException' );
-		new DispatchingSerializer( array( new \stdClass() ) );
+		new DispatchingSerializer( [ new \stdClass() ] );
```

Comment:
In cases like this where `stdClass` is only used as an *invalid* value I'm fine with it not appearing in the `use` section.